components/Bookmarks/BookmarkMiniFeed.js

```diff
@@ -10,7 +10,8 @@ import Link from '../Link/Path'
 import { getCollectionItems, registerQueryVariables } from './queries'
 
 const variables = {
-  collections: ['bookmarks']
+  collections: ['progress', 'bookmarks'],
+  progress: 'UNFINISHED'
```

Comment:
should be moved into a wrapper around the whole thing which does a different query based on `progressConsent`:
```
  const hasProgress = !!(me?.progressConsent)
  const variables = useMemo(() => {
    if (hasProgress) {
        return {
          collections: ['progress', 'bookmarks'],
          progress: 'UNFINISHED',
          lastDays: 30
        }
    }
    return {
      collections: ['bookmarks']
    }
  }, [hasProgress])
  registerQueryVariables(variables)
  return <BookmarkMiniFeed variables={variables} />
```

And then use a [options function](https://www.apollographql.com/docs/react/api/react/hoc/#configoptions) in the `graphql` wrapper:
```
  graphql(getCollectionItems, {
    options: props => ({
      fetchPolicy: 'cache-and-network',
      variables: props.variables
    })
  })
```